Core/DISET/private/InnerRPCClient.py

```diff
@@ -5,26 +5,26 @@
 
 from DIRAC.Core.DISET.private.BaseClient import BaseClient
 from DIRAC.Core.Utilities.ReturnValues import S_OK
-
+from DIRAC.Core.Utilities.DErrno import cmpError, ENOAUTH
```

Comment:
I don't remember exactly what was the problem with extension, but I think it is because I wanted to automate the loading of the extension error codes. I then gave up, and made it a function, called in all the dirac-(service|agent) and other intialization scripts.